_episodes_rmd/13-r-beyond-base-r.Rmd

```diff
@@ -147,19 +154,19 @@ Take a couple of minutes to look over the code and read the associated comments,
 
 ```{r}
```

Comment:
you may want to set `eval=FALSE` for this, so the plot files don't get generated during the rendering process.